lib/src/components/NxVulnerabilityIndicator/types.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { HTMLAttributes, WeakValidationMap } from 'react';
+import * as PropTypes from 'prop-types';
+
+export interface Props extends Omit<HTMLAttributes<HTMLDivElement>, 'className' | 'children'> {
+  criticalCount: number;
+  severeCount: number;
+  moderateCount: number;
+  lowCount: number;
+  layout?: string;
+  className?: string | null;
+}
+
+export const propTypes: WeakValidationMap<Props> = {
```

Comment:
Use `ValidationMap` instead of `WeakValidationMap` if possible

Reply:
> Use `ValidationMap` instead of `WeakValidationMap` if possible

I will confess to not fully understanding the difference: https://github.com/sonatype/sonatype-react-shared-components/pull/343/commits/ee516df5ef2ccaff199b3c2cb9b519c3fa563fc8

Reply:
> I will confess to not fully understanding the difference: ee516df

One actually works (meaning gives errors if your propTypes and TS types don't match) and the other doesn't seem to